tests/support_server/setup.pm

```diff
@@ -490,6 +491,29 @@ sub setup_mariadb_server {
     $disable_firewall = 1;
 }
 
+sub setup_nfs_server {
+    my $nfs_mount       = "/nfs/shared";
+    my $nfs_permissions = "rw,sync,no_root_squash";
+
+    # Added as the client test code might want to change the default
+    # values
+    if (get_var("CONFIGURE_NFS_SERVER")) {
+        $nfs_mount       = get_required_var("NFS_MOUNT");
+        $nfs_permissions = get_required_var("NFS_PERMISSIONS");
+    }
+
+    zypper_call('in rpcbind nfs-kernel-server');
+    systemctl("start rpcbind");
+    systemctl("start nfs-server");
+    assert_script_run("nfsstat –s");
+    assert_script_run("mkdir -p $nfs_mount");
+    assert_script_run("chmod 777 $nfs_mount");
+    assert_script_run("echo $nfs_mount 10.0.2.2/24\\($nfs_permissions\\) >> /etc/exports");
```

Comment:
Maybe it is better to to use `*` instead of  `10.0.2.2/24` or it should be `10.0.2.0/24` just to make more clear that whole network is defined, `10.0.2.2/24` could be confusing.

Reply:
I tried to use * and ?
Did not really work as expected with the IP. I read */? works only for fqdns, otherwise one might expect some misbehaviors.

Reply:
well, you can go the route in the middle and allow an ip network, e.g. 10.0.2.0/24